packages/comet-uswds/src/components/time-picker/time-picker.tsx

```diff
@@ -48,22 +70,33 @@ export const TimePicker = ({
   const timePickerRef = useRef<HTMLDivElement>(null);
   useLayoutEffect(() => {
     const timePickerElement = timePickerRef.current as HTMLDivElement;
-    timePicker.on(timePickerElement);
-    comboBox.on(timePickerElement);
-    const inputElement = timePickerElement.querySelector(
-      '.usa-combo-box__input',
-    ) as HTMLInputElement;
+    const comboBoxLabel = timePickerElement?.parentNode?.querySelector(`label[for="${id}"]`);
+
+    // Only intialize the combo box if the label is present, USWDS will error otherwise
+    if (comboBoxLabel) {
+      timePicker.on(timePickerElement);
+      comboBox.on(timePickerElement);
 
-    if (onChange) {
-      inputElement.addEventListener('change', onChange);
+      const inputElement = timePickerElement.querySelector(
+        '.usa-combo-box__input',
+      ) as HTMLInputElement;
+      if (onChange) {
+        inputElement.addEventListener('change', onChange);
+      }
     }
 
     return () => {
-      if (onChange) {
-        inputElement.removeEventListener('change', onChange);
+      if (comboBoxLabel) {
+        comboBox.off(timePickerElement);
+        timePicker.off(timePickerElement);
+
+        const inputElement = timePickerElement.querySelector(
+          '.usa-combo-box__input',
+        ) as HTMLInputElement;
+        if (onChange) {
+          inputElement.removeEventListener('change', onChange);
+        }
       }
-      comboBox.off(timePickerElement);
-      timePicker.off(timePickerElement);
     };
```

Comment:
Maybe update the display format from 8:32am to 8:32 a.m.

Reply:
This is using the format provided by USWDS so I htink we should leave it as is